client/stylesheets/_toCat.scss

```diff
@@ -5724,6 +5730,10 @@ textarea {
     color: #666 !important;
 }
 
+.headerNav > ul > li > a, .headerNav > ul > li > a:visited, .headerNav > ul > li > a:link {
+  color: #404040 !important;
```

Comment:
Color literals like `#404040` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used